changelog.d/3752.feature

```diff
@@ -0,0 +1 @@
+Better expose adding spaces as Subspaces #3752
```

Comment:
No need to add "#3752" it will be added by towncrier since it's in the filename

Reply:
done